web/gui/dashboard/src/domains/chart/utils/get-portal-node-styles.ts

```diff
@@ -0,0 +1,34 @@
+import { Attributes } from "./transformDataAttributes"
+import { ChartLibraryConfig } from "./chartLibrariesSettings"
+
+type GetPortalNodeStyles = (attributes: Attributes, chartSettings: ChartLibraryConfig) => {
+  height: string | undefined,
+  width: string | undefined,
+  minWidth: string | undefined
+}
+export const getPortalNodeStyles: GetPortalNodeStyles = (attributes, chartSettings) => {
+  let width
```

Comment:
It may seem as a lot of boilerplate work for a very tiny profit, yet it may be useful to abstract the let/if   procedures into immutable getters, with R.cond or something like that.

Reply:
good point. i was leaving code similar to old dashboard, because i thought it will be easy to compare and eventually find bugs (for example here: https://github.com/netdata/netdata/blob/6f3007537f1e4214bd5635d26f17c3d98e78afa6/web/gui/dashboard.js#L6506-L6522) but now i think most people reading new code won't even know the history of it, so it's better to write clean from the beginning